FaderPlugin/Plugin.cs

```diff
@@ -34,39 +39,44 @@ public class Plugin : IDalamudPlugin
     [PluginService] public static ITargetManager TargetManager { get; set; } = null!;
     [PluginService] public static IDataManager Data { get; private set; } = null!;
 
+    // Configuration and windows.
     public readonly Configuration Config;
+    private readonly WindowSystem _windowSystem = new("Fader");
+    private readonly ConfigWindow _configWindow;
 
-    private readonly WindowSystem WindowSystem = new("Fader");
-    private readonly ConfigWindow ConfigWindow;
+    // State maps and timers.
+    private readonly Dictionary<State, bool> _stateMap = new();
+    private bool _stateChanged;
+    private readonly Timer _chatActivityTimer = new();
+    private bool _hasChatActivity;
 
-    private readonly Dictionary<State, bool> StateMap = new();
-    private bool StateChanged;
-
-    // Idle State
-    private readonly Timer IdleTimer = new();
-    private bool HasIdled;
-
-    // Chat State
-    private readonly Timer ChatActivityTimer = new();
-    private bool HasChatActivity;
+    // Opacity Management
+    private readonly Dictionary<string, float> _currentAlphas = new();
+    private readonly Dictionary<string, bool> _finishingHover = new();
 
     // Commands
     private const string CommandName = "/pfader";
-    private bool Enabled = true;
+    private bool _enabled = true;
+
+    // Territory Excel sheet.
+    private readonly ExcelSheet<TerritoryType> _territorySheet;
 
-    private readonly ExcelSheet<TerritoryType> TerritorySheet;
+    // Delay management Utility
+    private readonly DelayManager _delayManager = new DelayManager();
+
+    // Enum Cache
+    private static readonly Element[] AllElements = (Element[])Enum.GetValues(typeof(Element));
```

Comment:
Not a full review yet as i currently lack time, but i noticed this while looking over the emails.

Please use the `<Element>` syntax here.
For whatever reason has `typeof(Element)` lead to some issues in a different plugin of mine, must have forgotten to change it here in Fader.